lib/generation/generators/value_objects/generation_configuration/pb_platform_orientation_generation_mixin.dart

```diff
@@ -25,7 +25,9 @@ mixin PBPlatformOrientationGeneration {
           formatedName,
           '${formatedName}_platform_builder.dart',
         ));
-    if (platformsMap.length > 1) {
+    if (platformsMap.length > 1 ||
+        (platformsMap.containsKey('mobile') &&
```

Comment:
To make this a bit more dynamic, what do you think of doing something like `platformsMap.values.any((orientations) => orientations.length > 1)`. This way, no matter the platform, we'll generate the platform builder if it has more than one platform 🤔 . I'll make the change, but feel free to let me know if you see an issue with it.

Reply:
Sure, that makes sense, you can go ahead & push the changes